### PR TITLE
bugfix: ZENKO-1004 Unskip one-to-many tests

### PR DESCRIPTION
**Pull request by bennettbuchanan**
Un-skip one-to-many tests and remove duplicate test cases.

### CODE REVIEW

File: tests/zenko_tests/node_tests/backbeat/ReplicationUtility.js [hunk 1]
```diff
@@ -725,7 +725,8 @@ class ReplicationUtility {
             }
             const { srcData, destMetadata, destData } = data;
             assert.strictEqual(srcData.ReplicationStatus, 'COMPLETED');
-            assert.strictEqual(srcData.ContentLength, destMetadata[0].size);
+            assert.strictEqual(`${srcData.ContentLength}`,
```
Comment (JianqinWang) [inline tests/zenko_tests/node_tests/backbeat/ReplicationUtility.js]: Why are we changing this?

Comment (rahulreddy) [reply tests/zenko_tests/node_tests/backbeat/ReplicationUtility.js]: I believe it was because the SDK changed the output from number to string

**Review by JianqinWang — APPROVED** [review]

**Review by rahulreddy — APPROVED** [review]

**Pull request merged**